main.go

```diff
@@ -97,7 +98,20 @@ func main() {
 		}
 	}
 
-	cli, err := NewCli(opts.ProjectId, opts.InstanceId, opts.DatabaseId, opts.Prompt, opts.HistoryFile, cred, os.Stdin, os.Stdout, os.Stderr, opts.Verbose, priority, opts.Role, opts.Endpoint, directedRead, opts.SkipTLSVerify)
+	// Don't need to unmarshal into descriptorpb.FileDescriptorSet because the UpdateDDL API just accepts []byte.
+	var protoDescriptorFileContent []byte
+	if opts.ProtoDescriptorFile != "" {
+		var err error
+		b, err := os.ReadFile(opts.ProtoDescriptorFile)
```

Comment:
I think we don't need to use temporal variable `b` here as `err` is defined above.

```
protoDescriptorFileContent, err = os.ReadFile(opts.ProtoDescriptorFile)
```

Reply:
Thank you! 
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/195/commits/30e2eabf6878dac74f75e77d083c42fc1d8ce8b6